pkg/cmd/init/options.go

```diff
@@ -247,18 +256,18 @@ func (o *TemplatesOptions) Run() error {
 	return nil
 }
 
-// getURL parses url from args, called when --online is true.
-func getURL(args []string) string {
+// templateRepoPathFromURL parses url from args, called when --online is true.
+func templateRepoPathFromURL(args []string) string {
```

Comment:
no need to change, `getURL` works for online mode only

Reply:
templateRepoFromOnline is better, thanks :)